packages/hoverable/src/with-hover.spec.tsx

```diff
@@ -98,4 +99,79 @@ describe('withHover HOC', () => {
             expect(nestedElement).not.toHaveStyle(nestedHoverStyle);
         });
     });
+    describe('Children props', () => {
```

Comment:
Maybe it'll be more explicit if you merge with previous describe so the structure will be like this
```describe('Behaviour with children'() => {
            describe('Children styles', () => {...});
            describe('Children props', () => {...});
})```